test/test_corpus.cc

```diff
@@ -132,6 +132,56 @@ unique_ptr<Diagnostic> errorToDiagnostic(const core::GlobalState &gs, const core
     return make_unique<Diagnostic>(Range::fromLoc(gs, error.loc), error.header);
 }
 
+class ErrorHandler {
+    vector<unique_ptr<core::Error>> &errors;
+    Expectations &test;
```

Comment:
(A reference to expectations makes sense, as it's a global entity whose lifetime is linked w/ the test.)